packages/cli-repl/test/fixtures/simple-console-log.js

```diff
@@ -1,2 +1 @@
-// eslint-disable-next-line no-console
```

Comment:
Is `no-console` no longer a default rule? It’s a pretty good one, if you ask me :)

Reply:
Yeah it did give me pause. I think I'll just add it back.

Reply:
https://jira.mongodb.org/browse/MONGOSH-1505